doc/dynamic-port-breakout/sonic-dynamic-port-breakout-HLD.md

```diff
@@ -1190,6 +1192,73 @@ All thee attribute that coud be changed in orchagent should be able to be brough
 # Warm reboot support
 Syncd changes are required as mentioned above. The PR need to be merged and tested.
 
+# Port breakout workflow additions 
+xcvrd (transceiver daemon) running as part of PMON docker container, detects optical module (transceiver) presence.
+If transceiver is found as QSFP-DD, it initiates and orchestrates entire CMIS FSM until module ready state.
+As part of enabling CMIS FSM with port breakout, found out that port breakout feature is not supported for QSFP-DD optical modules.
+
+**Few key things to take into account prior to getting into workFlow**
+
+A NxS breakout cable inserted implies following
+- A physical port is broken down into N sub-ports (logical ports)
+  - sub-ports are numbered as 8/N i.e.
+  - For 4x100G breakout optical module inserted in physcial Ethernet1 port, implies: Ethernet8, Ethernet10, Ethernet12, Ethernet14
+  - Note: This is done is this manner to keep such assignments uniform across various breakout modes viz.1x, 2x, 4x, 8x
+- Speed of each sub-port is S Gpbs
+- Unique channel# is assigned to each of the N sub-port starting with channel# 1 (and sequentially incrementing with each sub-port)
+
+**Following is the new design (workflow) for 'port breakout' to work end-to-end with 'CMIS enabled' (in SONiC):**
+
+!['port breakout feature workflow with CMIS'(1)](https://user-images.githubusercontent.com/69485234/226458948-f059bda7-589f-48ab-9e89-3f5b353d93fc.png)
+
+['port breakout feature workflow with CMIS'.pdf](https://github.com/shyam77git/SONiC/files/11022820/port.breakout.feature.workflow.with.CMIS.pdf)
+
+
+- Configure a unique channel # for each broken-down (logical) port in platform's port_config.ini
+   - channel # to start with 1 (and increment sequentailly for each logical port) under the same physcial port
```

Comment:
I strongly suggest NOT using 'channel' here. This part of change will be in XCVRD, which is mainly dealing with optical cables.  In the optical cables area, 'channel' is a concept which already widely used and has a clear definition(open any SFF or CMIS specification we can see it),  we even have SONiC platform APIs defined to handle optical cable 'channels'.  

Use 'channel' to refer to a 'sub-port' is misleading and will cause confusion.

Reply:
Per the discussion in the changeset PR, I'd update 'channel' terminology in this HLD to refer them as 'subport'

Reply:
Updated in latest update (PR commit)